src/main/java/car/domain/Car.java

```diff
@@ -1,19 +1,26 @@
-package car;
+package car.domain;
 
 import java.util.Random;
 
 public class Car {
 
     private int position = 0;
     private String name;
+    private Movable  movable;
 
     private Car(){}
     public Car(String name){
         carNameCheck(name);
         this.name = name;
     }
-    public void move(int moveStrategy){
-        if(moveStrategy > 3) {
+
+    private void setMovable(Movable movable){
+        this.movable = movable;
+    }
+
+    public void move(Movable movable){
+        setMovable(movable);
+        if(movable.moveStrategy() > 3) {
             position += 1;
         }
     }
```

Comment:
전략 객체를 파라미터로 전달 받기 때문에 setter를 제거할수 있을것 같아요. 
추가로 더이상 Car 클래스는 Movable 객체를 가지고 있지 않아도 될것 같은데 어떻게 생각하시나요 ? 
```suggestion
    public void move(Movable movable){
        if(movable.moveStrategy() > 3) {
            position += 1;
        }
    }
```